exe/cli/cmd/wallet_new.go

```diff
@@ -0,0 +1,61 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/caarlos0/spin"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	newCmd.Flags().StringP("type", "t", "bls", "specifies the wallet type, either bls or secp256k1. Defaults to bls.")
+	newCmd.Flags().BoolP("switch", "s", false, "whether or not to switch to the newly created wallet")
+
+	walletCmd.AddCommand(newCmd)
+}
+
+var newCmd = &cobra.Command{
+	Use:   "new",
+	Short: "Create a new filecoin wallet",
+	Long:  `Create a new filecoin wallet`,
```

Comment:
`s/`wallet`/`wallet address`

Or maybe we should say simply _address_ and avoid _wallet_?
Or maybe wallet is OK despite isn't what is happening under the hoods. A single-address wallet.